Samples/AlgorithmsWarps/Program.cs

```diff
@@ -46,30 +46,25 @@ static void KernelWithWarpExtensions(ArrayView2D<int, Stride2D.DenseX> data)
         static void Main()
         {
             // Create default context and enable algorithms library
-            using (var context = Context.Create(builder => builder.Default().EnableAlgorithms()))
+            using var context = Context.Create(builder => builder.Default().EnableAlgorithms());
+
+            // For each available device...
+            foreach (var device in context)
             {
-                // For each available device...
-                foreach (var device in context)
-                {
-                    // Create the associated accelerator
-                    using (var accelerator = device.CreateAccelerator(context))
-                    {
-                        Console.WriteLine($"Performing operations on {accelerator}");
+                // Create the associated accelerator
+                using var accelerator = device.CreateAccelerator(context);
+                Console.WriteLine($"Performing operations on {accelerator}");
 
-                        var kernel = accelerator.LoadStreamKernel<ArrayView2D<int, Stride2D.DenseX>>(KernelWithWarpExtensions);
-                        using (var buffer = accelerator.Allocate2DDenseX<int>(new LongIndex2D(accelerator.WarpSize, 4)))
-                        {
-                            kernel((1, buffer.IntExtent.X), buffer.View);
-                            accelerator.Synchronize();
+                var kernel = accelerator.LoadStreamKernel<ArrayView2D<int, Stride2D.DenseX>>(KernelWithWarpExtensions);
+                using var buffer = accelerator.Allocate2DDenseX<int>(new LongIndex2D(accelerator.WarpSize, 4));
+                kernel((1, buffer.IntExtent.X), buffer.View);
+                accelerator.Synchronize();
```

Comment:
nit: should we remove the explicit synchronization here and place a note that the synchronization happens implicitly by a call to `GetAsArray2D()`?